proto/desmos/profiles/v3/models_chain_links.proto

```diff
@@ -115,28 +119,87 @@ message HexAddress {
   string prefix = 2 [ (gogoproto.moretags) = "yaml:\"prefix\"" ];
 }
 
-// SingleSignatureData is the signature data for a single signer
-message SingleSignatureData {
+// --------------------------------------------------------------------------------------------------------------------
+
+// CosmosSingleSignature is the signature data for a single signer
+message CosmosSingleSignature {
```

Comment:
I'm thinking if there are better ways to name these type after what @dadamu said: 

> Luckily, thanks to [solana-wallet-adapter](https://github.com/solana-labs/wallet-adapter) providing the function signing a raw message(arbitrary string), we can use the same process as CosmosSingleSignature [...].

This means that what has been currently named `CosmosSingleSignature` is actually compatible with other ecosystems different from the Cosmos one. For this reason, I think a better name should be used instead. 

@dadamu Proposed `GeneralSingleSignature`, but maybe even a shorter `SingleSignature` can work. 

What do you think @bragaz @kwunyeung? 

**Note**  
By looking at the code, we know that the signing algorithm used to verify the signature will be based on the type of the private key used. The currently supported public key types are `Ed25519`, `Secp256r1` and `Secp256k1`. As @dadamu said, for Solana we should use `/cosmos.crypto.ed25519.PubKey` (hence why this signature also works for that ecosystem).

Reply:
I think we can use something like: `StandardSingleSignature` or `GeneralSingleSignature` is ok. Then we can add in the docs the supported chains of such signature

Reply:
@RiccardoM From my side, the `General` is compared to `EVM` or other unique spec signatures. Noting that `SingleSignature` is the general signature based on general private/public key signing should be enough as well.

Reply:
I ended up renaming `CosmosSingleSignature` to `SingleSignature`, and removing `EVMSignature` in favor or `SingleSignature` as well. Now EVM-compatible signatures can be specified by simply using the proper `SignatureValueEncoding` inside a `SingleSignature` itself.